packages/backend/server/src/plugins/copilot/providers/anthropic.ts

```diff
@@ -28,15 +32,28 @@ export type AnthropicConfig = {
   baseUrl?: string;
 };
 
-export class AnthropicProvider
-  extends CopilotProvider<AnthropicConfig>
-  implements CopilotTextToTextProvider
-{
+export class AnthropicProvider extends CopilotProvider<AnthropicConfig> {
   override readonly type = CopilotProviderType.Anthropic;
-  override readonly capabilities = [CopilotCapability.TextToText];
   override readonly models = [
-    'claude-3-7-sonnet-20250219',
-    'claude-3-5-sonnet-20241022',
+    {
+      id: 'claude-3-7-sonnet-20250219',
+      capabilities: [
+        {
+          input: [ModelInputType.Text],
```

Comment:
[ModelInputType.Text, ModelInputType.Image]

Reply:
Need to check all models for allowed input and output types. Is there any official instructions for this?

Reply:
done